### DISCUSSION TITLE
Java implementation of Table Driven with lot of columns

### DISCUSSION THREAD

**Discussion by thomasanup** [getgauge/gauge#2178] [General]
In the spec file :

table: data/entities/csv/data1.csv
`* Add data <col1> <col2> <col3>.........<colN>.`

Java:

In the java implementation function  ,  I   have a col1 to colN as parameters like 
public void addData(String col1, String col2 ....String colN)

Is there a way I can pass the entire csv row as an object to my java program

**Comment by sriv** [comment]
you can have nested tables (i.e. a table as a cell within another table). Other than that, it will have to be either a full table or a cell, I'm afraid.
